### PR TITLE
Update 13-hosting.md

### PR DESCRIPTION
**Pull request by pshellito**
Tried to fix broken link to page source... still not working. I shouldn't have committed yet, I'll take another look (I'm new to this)...

### CODE REVIEW

File: _episodes/13-hosting.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ communities already using the same service.
 
 As an example, Software Carpentry [is on
 GitHub](https://github.com/swcarpentry/) where you can find the [source for this
-page](https://github.com/swcarpentry/git-novice/blob/gh-pages/13-hosting.md).
+page](https://github.com/swcarpentry/git-novice/blob/gh-pages/_episodes/13-hosting.md).
```
Comment (iglpdc) [inline _episodes/13-hosting.md]: Good catch! But we prefer to use the variables defined in `_config.yml` for this type of URLs, so we don't get broken links so when we move things around :)

`{{ page.root }}` equals the root dir of the repo. So you should do `...[source for this page]({{page.root}}/_episodes/13-hosting.md`

File: _episodes/13-hosting.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ communities already using the same service.
 
 As an example, Software Carpentry [is on
 GitHub](https://github.com/swcarpentry/) where you can find the [source for this
```
Comment (iglpdc) [inline _episodes/13-hosting.md]: You could also change this one, if you don't mind: `https://github.com/swcarpentry`-> ` {{ swc_github }}` (see again `_config.yml`.

**Comment by iglpdc** [comment]
Thanks @pshellito !

**Comment by pshellito** [comment]
@iglpdc looks like it's still not working, unfortunately. Maybe it just need some time to re-run jekyll.

**Pull request merged**